src/translations/en/checkoutForm.json

```diff
@@ -18,6 +18,7 @@
     "for": "for",
     "interval": "{{count}} {{interval}}",
     "interval_plural": "{{count}} {{interval}}s",
-    "tax": "+ Tax"
+    "tax": "+ Tax",
+    "removeCoupon": "REMOVE"
```

Comment:
I think you forgot the french translation

Reply:
Thanks good catch